### PR TITLE
COM-3259 Added quote email templates

### PR DESCRIPTION
**Pull request by SharmaBhushan**

### CODE REVIEW

File: labels/de-DE.json [hunk 1]
```diff
@@ -369,7 +369,7 @@
   "orderNo": "Order #",
   "orderNumber": "Bestellnummer",
   "orderPickupBlob": "Bonne nouvelle! <br/> Votre commande est disponible et prête à être récupérée. Veuillez cliquer sur le bouton «Je suis en route» ci-dessous pour informer le magasin que vous êtes sur le point de récupérer votre commande. Merci beaucoup pour votre achat! nous sommes heureux de pouvoir vous aider à obtenir ce que vous recherchez.",
-  "orderPickupReminderTitle": "RAPPEL: VOTRE COMMANDE EST PRÊT POUR LE RETRAIT!", 
+  "orderPickupReminderTitle": "RAPPEL: VOTRE COMMANDE EST PRÊT POUR LE RETRAIT!",
```
Comment (sagarraut0007) [inline labels/de-DE.json]: Remove this change from PR. since you have not changed this label.

File: labels/en-US.json [hunk 2]
```diff
@@ -460,7 +460,7 @@
   "orderPickupBlob": "Good News!<br/>Your order is available and ready for pick up. Please click the \"I'm on my way\" button below to inform the store you are on your way to pick up your order. Thank you so much for your purchase! we're happy we can help you get what you're looking for.",
   "orderPickupReadyBlob1": "Thank you for shopping at {0}. Your order #{1}, external order #{2} is ready for pickup.",
   "orderPickupReadyBlob2": "Important notes about your order: This order can only be picked up by the owner of the credit card used to make the purchase or the pickup party designated in checkout. Bring a government-issued ID with your picture on it (such as a driver's license or passport). You may continue to receive email reminders until you get your order.",
-  "orderPickupReminderTitle": "REMINDER: YOUR ORDER IS READY FOR PICK-UP!",
```
Comment (sagarraut0007) [inline labels/en-US.json]: Remove this change from PR. since you have not changed this label.

File: templates/email/quote-expired.hypr [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+         <!--- Header --->
+            <div class="mz-curbside-email-header">  
+            <img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+            <div class="mz-curbside-header-body">
+                <p><strong>{{ siteContext.generalSettings.websiteName }}</strong></p>
+                <h3>{{ labels.curbsideOrderConfirmationtxt }}</h3>
+            </div>
+            </div>
+           
+<div style="margin: 3% 0;">
```
Comment (sagarraut0007) [inline templates/email/quote-expired.hypr]: Remove inline styling and create classes and use it here. Do this for all templates.

**Review by sagarraut0007 — CHANGES_REQUESTED** [review]

File: labels/de-DE.json (hunk 1 above)
Comment (SharmaBhushan) [reply labels/de-DE.json]: Done

File: labels/en-US.json (hunk 2 above)
Comment (SharmaBhushan) [reply labels/en-US.json]: Done

File: templates/email/quote-expired.hypr (hunk 3 above)
Comment (SharmaBhushan) [reply templates/email/quote-expired.hypr]: Remove inline css and added "BR" tag for manage text content space

File: labels/en-US.json (hunk 2 above)
Comment (sagarraut0007) [reply labels/en-US.json]: I'm still seeing this issue. Can you please fix it.

**Review by sagarraut0007 — CHANGES_REQUESTED** [review]

File: labels/en-US.json (hunk 2 above)
Comment (SharmaBhushan) [reply labels/en-US.json]: Done

File: labels/de-DE.json [hunk 4]
```diff
@@ -594,5 +594,10 @@
   "width": "Breite",
   "wrongCountryForType": "Bitte wählen Sie ein Land für diesen Kontakt Typ unterstützt",
   "yourAccount": "Ihr Konto",
-  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>."
+  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>.",
+  "quoteInReview": "Bonjour, <br /> <br /> <br /> Le numéro du devis: {0} et le nom du devis: {1} ont été mis à jour. Veuillez examiner les modifications.",
+  "quoteReadyCheckout": "Bonjour, <br /> <br /> <br /> Le numéro du devis: {0} nom du devis: {1} a été mis à jour. Veuillez consulter les mises à jour en vous connectant à votre compte.",
```
Comment (rakesh-dontula) [inline labels/de-DE.json]: ReadyForCheckout.. change it everywhere..

File: theme.json [hunk 5]
```diff
@@ -271,6 +271,24 @@
...
       "title": "Customer In Transit"
+    },
+    {
+      "id": "quote.inreview",
+      "properties": {},
+      "template": "email/quote-in-review",
+      "title": "Quote In Review"
+    },
+    {
+      "id": "quote.readycheckout",
```
Comment (rakesh-dontula) [inline theme.json]: `quote.readyforcheckout`

**Review by rakesh-dontula — CHANGES_REQUESTED** [review]

File: labels/de-DE.json (hunk 4 above)
Comment (SharmaBhushan) [reply labels/de-DE.json]: Done

File: theme.json (hunk 5 above)
Comment (SharmaBhushan) [reply theme.json]: Done

File: labels/de-DE.json [hunk 6]
```diff
@@ -594,5 +594,12 @@
   "width": "Breite",
   "wrongCountryForType": "Bitte wählen Sie ein Land für diesen Kontakt Typ unterstützt",
   "yourAccount": "Ihr Konto",
-  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>."
+  "yourOrderNumber": "Ihre Auftragsnummer ist <strong> {0} </ strong>.",
```
Comment (ghost) [inline labels/de-DE.json]: The localization file is `DE-de` which is for German. Your text is in French. Please update this to be German as it should be. Ignore that some other labels are incorrectly in French.

You should also add the labels in alphabetical order in the right place in the file. The label `quoteInReview` should be after `quoteHistory` for example. Please add the labels in the right place. Thanks.

File: labels/en-US.json [hunk 7]
```diff
@@ -460,7 +460,7 @@
   "orderPickupBlob": "Good News!<br/>Your order is available and ready for pick up. Please click the \"I'm on my way\" button below to inform the store you are on your way to pick up your order. Thank you so much for your purchase! we're happy we can help you get what you're looking for.",
   "orderPickupReadyBlob1": "Thank you for shopping at {0}. Your order #{1}, external order #{2} is ready for pickup.",
   "orderPickupReadyBlob2": "Important notes about your order: This order can only be picked up by the owner of the credit card used to make the purchase or the pickup party designated in checkout. Bring a government-issued ID with your picture on it (such as a driver's license or passport). You may continue to receive email reminders until you get your order.",
-  "orderPickupReminderTitle": "REMINDER: YOUR ORDER IS READY FOR PICK-UP!",  
+  "orderPickupReminderTitle": "REMINDER: YOUR ORDER IS READY FOR PICK-UP!",
```
Comment (ghost) [inline labels/en-US.json]: Revert unrelated changes. Only include changes you made for this story. Don't touch other labels.

File: templates/email/quote-expired.hypr [hunk 8]
```diff
@@ -0,0 +1,26 @@
+﻿{% extends "email/email" %}
+{% block body-content %}
+         <!--- Header --->
+            <div class="mz-curbside-email-header">  
+            <img src="https://kibocommerce.com/wp-content/themes/kibo/images/logo.png" alt="logo">
+            <div class="mz-curbside-header-body">
+                <p><strong>{{ siteContext.generalSettings.websiteName }}</strong></p>
+            </div>
+            </div>
```
Comment (ghost) [inline templates/email/quote-expired.hypr]: Why is this using the curbside header for a B2B email? Don't do that. Also we should be putting headers in a shared file and reusing it in the B2B templates. I don't know if we did that for the other B2B emails but we should be.

Also please format this file nicely. Proper tabbing/spacing, etc.

This applies to the other emails as well. Please fix all of them.

File: templates/email/quote-expired.hypr [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+           
+<br />
+<div>
+{% if model.IsSeller %}
+<br /><br />
+   {{ labels.quoteExpiredSeller|string_format(model.name, model.number)|safe }} {{ model.expirationDate|date('M d, Y') }}. {{ labels.pleaseReview }} <a href="//{{ domainName }}/selleraccount/quote/{{model.id}}">{{labels.hereLink}}</a>.
```
Comment (ghost) [inline templates/email/quote-expired.hypr]: This link won't work for sellers. We have to link to the seller domain and not the storefront website. Need to figure out the right URL and update this.

This applies to the other emails as well. Need to fix all of them.

File: labels/de-DE.json (hunk 6 above)
Comment (SharmaBhushan) [reply labels/de-DE.json]: Done.

File: labels/en-US.json (hunk 7 above)
Comment (SharmaBhushan) [reply labels/en-US.json]: Done

File: templates/email/quote-expired.hypr (hunk 8 above)
Comment (SharmaBhushan) [reply templates/email/quote-expired.hypr]: Done, make another css class 'mz-main-email-header' and using for all templates

File: templates/email/quote-expired.hypr (hunk 9 above)
Comment (SharmaBhushan) [reply templates/email/quote-expired.hypr]: Changed url: 
<pre>
{% if model.IsSeller %}
     <a href="//{{ domainName }}/Admin/s-{{model.siteId}}/b2baccounts/edit/{{model.customerAccountId}}">{{labels.hereLink}}</a>.
    {% else %}
     <a href="//{{ domainName }}/myaccount/quote/{{model.id}}">{{labels.hereLink}}</a>.
    {% endif %}
</pre>

File: templates/email/quote-expired.hypr (hunk 8 above)
Comment (SharmaBhushan) [reply templates/email/quote-expired.hypr]: Remove 'mz-main-email-header' and using 'mz-b2b-email-header' class for template

**Review by ghost — APPROVED** [review]
